tests/unit/amo/components/TestCollection.js

```diff
@@ -1240,51 +1248,7 @@ describe(__filename, () => {
       deleteCollectionAddonNotes({
         addonId,
         errorHandlerId: errorHandler.id,
-        page,
-        slug: collectionDetail.slug,
-        username: collectionDetail.author.username,
-      }),
-    );
-  });
-
-  it('dispatches deleteCollectionAddonNotes when deleteNote is called without a page defined', () => {
```

Comment:
As mentioned above, this test case was removed because there's no need to test the functionality `without a page defined`, because that no longer applies to this component.